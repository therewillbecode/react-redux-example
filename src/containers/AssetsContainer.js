import React from "react";
import { connect } from "react-redux";

import Assets from "./components/Assets";

export const AssetsContainer = ({ assets }) => <Assets assets={assets} />;

const mapStateToProps = state => {
  return {
    assets: visibleAssets(state)
  };
};

export default connect(mapStateToProps)(AssetsContainer);
