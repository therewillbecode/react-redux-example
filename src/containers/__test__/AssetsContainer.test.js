/* eslint-disable */
import React from "react";
import ReactShallowRenderer from "react-test-renderer/shallow";

import { AssetsContainer } from "../AssetsContainer";

describe("AssetsContainer", () => {
  it("renders correctly", () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<AssetsContainer />);
    const tree = renderer.getRenderOutput();
    expect(tree).toMatchSnapshot();
  });
});
