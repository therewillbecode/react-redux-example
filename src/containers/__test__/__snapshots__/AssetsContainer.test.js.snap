// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AssetsContainer renders correctly 1`] = `
<Assets
  assets={undefined}
/>
`;
